packages/datasource-hubspot-translation/src/hs-client.ts

```diff
@@ -23,10 +23,16 @@ export default class HubSpotClient {
       )}`,
     );
 
-    const { results } = await this.client.crm.objects.searchApi.doSearch(
-      apiName,
-      publicObjectSearchRequest,
-    );
+    let results = [];
```

Comment:
I have a little suggestion, I've seen guyz speaking about [throttling](https://community.hubspot.com/t5/APIs-Integrations/Rate-limits-API/m-p/1002704) calls, don't know if that would be a good idea: 
- Provide a parameter when you create a hubspot datasource to tell if hubspot data are mandatory to load collection
- If data is flagged as mandatory, throttle the calls

WDYT ?